keras/objectives.py

```diff
@@ -63,6 +63,36 @@ def cosine_proximity(y_true, y_pred):
     y_pred = K.l2_normalize(y_pred, axis=-1)
     return -K.mean(y_true * y_pred, axis=-1)
 
+# The actual pinball/ tilted loss function for quantile regression
+def tilted_loss(q, y, f):
```

Comment:
Syntax should be `tilted_loss(y_true, y_pred, quantile)`. Unclear that `titled_loss` is the optimal name here; how is it called in sklearn?